README.md

```diff
@@ -21,6 +21,10 @@ Install or update dependencies via Composer:
     # if you don't already have Composer installed:
     curl -sS https://getcomposer.org/installer | php
     php composer.phar update
+    cd ../..
+    cd plugins/paymethod/paypal
+    php ../../../composer.phar update
```

Comment:
Per these instructions, I think `composer.phar` will be in `lib/pkp`, not the OJS root. I'd actually recommend that we don't include installation instructions for Composer but instead assume it'll already be present. This aspect confuses our instructions with relative paths etc.

Also, it's possible to specify the working directory on the Composer command-line, e.g:

```
composer -d=lib/pkp update
composer -d=plugins/paymethod/paypal
```
...rather than doing all that path munging. (If you fix that here, please ping me and I'll add it to the various scripts that could also make use of it!)

Reply:
The update looks good but I think this last line is still off:

```
php ../../../composer.phar update
```

It should be:

```
php ../../../lib/pkp/composer.phar update
```

Reply:
I'd suggest getting rid of the relative path entirely, and treating Composer as though it's already installed (e.g. `composer update` rather than `php path/to/composer.phar update`).